toolchain/check/call.cpp

```diff
@@ -97,7 +97,7 @@ static auto PerformCallToGenericClass(Context& context, SemIR::LocId loc_id,
   auto callee_specific_id =
       ResolveCalleeInCall(context, loc_id, generic_class,
                           EntityKind::GenericClass, enclosing_specific_id,
-                          /*self_type_id=*/SemIR::InstId::None,
+                          /*self_type_id=*/SemIR::TypeInstId::None,
```

Comment:
Should `ResolveCalleeInCall`'s `self_type_id` parameter be changed to a `TypeInstId` also? Or is that a TODO for later?

Reply:
FWIW this one in interface.cpp:
```
    auto self_facet_id =
        GetSelfFacet(context, interface_specific_id,
                     context.functions().Get(fn->function_id).generic_id,
                     self_type_id, self_witness_id);
    return GetFunctionTypeWithSelfType(
        context, context.types().GetInstId(interface_fn_type_id),
        self_facet_id);
```